pydarn/plotting/fan.py

```diff
@@ -341,8 +341,8 @@ def plot_fan(cls, dmap_data: List[dict], ax=None, ranges: List = [],
         if ccrs is None:
             azm = np.linspace(0, 2 * np.pi, 100)
             r, th = np.meshgrid(rs, azm)
-            plt.plot(azm, r, color='k', ls='none')
-            plt.grid()
+            ax.plot(azm, r, color='k', ls='none')
+            ax.grid()
```

Comment:
```suggestion
```

I'm still getting the grid lines not being shown on fan plots and can only get them to show up by removing this line. Are they appearing on your fan plots?

FOV, Map and grids are all fine for me though, I think I removed this line in another branch too as it was stopping the grid lines... no idea why though.

Reply:
```suggestion
            ax.grid(True)
```